www/includes/utility.php

```diff
@@ -1083,19 +1083,19 @@ function _major_summary_title($major, $data, $LISTURL, $daytext) {
 function score_to_strongly($dmpscore) {
     $dmpdesc = "unknown about";
     if ($dmpscore > 0.95 && $dmpscore <= 1.0)
-        $dmpdesc = "very strongly against";
+        $dmpdesc = "consistently voted against";
     elseif ($dmpscore > 0.85)
-        $dmpdesc = "strongly against";
+        $dmpdesc = "almost always voted against";
```

Comment:
Interestingly, I personally consider "almost always" stronger than "consistently voted" (which to me means most of the time, but less than almost always would imply).